### DISCUSSION TITLE
When signing a custom template, nuclei fail to find the folder of the default tempaltes

### DISCUSSION THREAD

**Discussion by Greatbug333** [projectdiscovery/nuclei#4354] [General]
### Nuclei version:
nuclei 3.0.2

### OS
windows 10

### Problem description

When I use Signing Custom Template and -t is specified as the file name, nuclei returns success=0 failed =0.
But when -t is specified as  full path + file name, nuclei returns success=1 failed=0

![image](https://github.com/projectdiscovery/nuclei/assets/51732528/a9b3d5a2-71de-4efa-9fa5-7c73f3b3169d)

When I use -tv, nuclei find the default path.

![image](https://github.com/projectdiscovery/nuclei/assets/51732528/02a944a6-f93e-4a7a-8bbc-ae21b04b4736)

I think this is a BUG that nuclei can't find the default templates path.

**Comment by tarunKoyalwar** [comment]
@Greatbug333 , to avoid un-intentional signing of templates . sign utility requires that a valid template path should be given (absolute or relative) in `-t` and it will not sign any templates loaded using tags or when specifying paths like `http/` `http/cves/` etc

but you should still be able to sign if you use relative paths ex: if your template exist in same folder you are running nuclei you can do 
```console
$ nuclei.exe -t .\djp-test1.yaml -sign
```
please share if you are unable to do so 

Thanks !

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
